docs/reference/indices/seal.asciidoc

```diff
@@ -0,0 +1,74 @@
+[[indices-seal]]
+== Seal
+
+The seal API allows to flush one or more indices and adds a marker to primaries and replicas if there are no pending write operations.
+The seal marker is used during recovery after a node restarts. If a replica is allocated on a node which already has a shard copy with the same seal as the primary then no files will be copied during recovery.
+Sealing is a best effort operation. If write operations are ongoing while the sealing is in progress then writing the seal might fail on some copies.
+
+[source,js]
+--------------------------------------------------
+$ curl -XPOST 'http://localhost:9200/twitter/_seal'
+--------------------------------------------------
+
+The response contains details about for which shards a seal was written and the reason in case of failure.
+Response in case all copies of a shard successfully wrote the seal:
```

Comment:
yes, that makes sense. not exactly an expert in rest statuses though. Do you think it would be ok to return CONFLICT in case the operation failed? Also in case it only succeeded on some shards should we return PARTIAL_CONTENT or something of the likes?